### PR TITLE
Update the Main page

### PR DESCRIPTION
**Pull request by VictoriaXiao**

### CODE REVIEW

File: _layouts/home.html [hunk 1]
```diff
@@ -31,20 +31,21 @@
...
       </div>
       <div class="col-12 col-md-7 col-lg-6 order-1 order-md-2 d-flex align-items-end flex-column">
-        <h1 style="text-align: right;">载歌在谷春游活动</h1>
+        <h1 style="text-align: right;">2022志愿者招新</h1>
         <div style="margin: 10px 50px">
-          <h2>2022春游：</h2>
+          <h2>2022志愿者招新：</h2>
           <p>
-            时间：2022年4月17日周日，11AM - 3PM PDT <br>
+            时间：2022年05月01日周日，5:00AM - 7:00PM PDT <br>
```
Comment (zzhai) [inline _layouts/home.html]: 5:00PM

File: _layouts/home.html [hunk 2]
```diff
@@ -31,20 +31,21 @@
...
-&nbsp;&nbsp; - 补发奖状奖杯T恤<br>
+            歌手赛招新组: <br>
+&nbsp;&nbsp; - 宣传组<br>
+&nbsp;&nbsp; - 技术组<br>
+&nbsp;&nbsp; - 赞助组<br>
+&nbsp;&nbsp; - 摄制组<br>
+&nbsp;&nbsp; - 内联组<br>
             扫码报名: 
-            <img style="height: 80px; width: 80px; margin:10px 0 10px 20px" src="https://res.cloudinary.com/zaigezaigu/image/upload/v1649533579/zgzg-io-website/2022%E6%98%A5%E6%99%9A%E6%8B%8D%E6%91%84%E8%8A%B1%E7%B5%AE/spring_hks66h.png" />
+            <img style="height: 80px; width: 80px; margin:10px 0 10px 20px" src="https://mmbiz.qpic.cn/sz_mmbiz_png/L3FiccBk78Xt1X2f1Dx7HeCsPY6XpGM6QfjaObWhib46qyjgrrr096mx1C21fqn6vVRTWaCjPJk1mM19LicyfsQibQ/640?wx_fmt=png&amp;wxfrom=5&amp;wx_lazy=1&amp;wx_co=1" />
```
Comment (zzhai) [inline _layouts/home.html]: Will send you the Cloudinary (a pic hosting site) account.

Comment (VictoriaXiao) [reply _layouts/home.html]: Upload the QR code to the Cloudinary

File: _layouts/home.html [hunk 3]
```diff
@@ -31,20 +31,21 @@
...
       <div class="col-12 col-md-7 col-lg-6 order-1 order-md-2 d-flex align-items-end flex-column">
-        <h1 style="text-align: right;">载歌在谷春游活动</h1>
+        <h1 style="text-align: right;">2022志愿者招新</h1>
         <div style="margin: 10px 50px">
-          <h2>2022春游：</h2>
+          <h2>2022志愿者招新：</h2>
           <p>
-            时间：2022年4月17日周日，11AM - 3PM PDT <br>
+            时间：2022年05月01日周日，5:00PM - 7:00PM PDT <br>
             地点：40000 Paseo Padre Pkwy, Fremont, CA 94538<br>
```
Comment (zzhai) [inline _layouts/home.html]: Please put in the new address.

**Review by zzhai — APPROVED** [review]

**Pull request merged**